mirrord/cli/src/config.rs

```diff
@@ -217,3 +221,29 @@ pub(super) enum OperatorCommand {
         namespace: OperatorNamespace,
     },
 }
+
+#[derive(ValueEnum, Debug, Clone)]
+pub enum Format {
+    Json,
+}
+
+#[derive(Args, Debug)]
+pub(super) struct LsArgs {
+    /// Specify the format of the output.
+    #[arg(
+        short = 'o',
+        long = "output",
+        num_args = 1,
+        value_name = "FORMAT",
+        value_enum
+    )]
+    pub output: Option<Format>,
+
+    /// Prettify the output.
+    #[arg(short = 'p', long)]
+    pub prettify: bool,
```

Comment:
I don't think we need this, if anything this should be another format. (I can't see how this applies to table for example)

Reply:
okay, do you want me to remove it?